omod/src/test/java/org/openmrs/module/owa/filter/OwaFilterTest.java

```diff
@@ -106,7 +106,7 @@ public void testOwaFilterUsesGlobalProperty() throws Exception {
 		req = new MockHttpServletRequest("GET", "openmrs/index.htm");
 		req.setServletPath("/index.htm");
 		owaFilter.doFilter(req, rsp, mockFilterChain);
-		Assert.assertEquals(rsp.getStatus(), 200);
```

Comment:
Can you explain a bit why we had to change from 200 to 302?

Reply:
Redirects give the status code 302. I tested this on the platform.

![Screenshot from 2024-06-05 14-59-22](https://github.com/openmrs/openmrs-module-owa/assets/61653572/277772b1-2ead-40e4-9e9c-67a06d09ed14)

Reply:
Did that change starting from platform 1.11.0?